java/src/jmri/jmrit/operations/locations/Track.java

```diff
@@ -71,37 +77,55 @@ public class Track extends PropertyChangeSupport {
+    protected String _destinationOption = ALL_DESTINATIONS; // track destination
+                                                            // restriction
     protected List<String> _destinationIdList = new ArrayList<>();
 
     // schedule options
     protected String _scheduleName = NONE; // Schedule name if there's one
     protected String _scheduleId = NONE; // Schedule id if there's one
     protected String _scheduleItemId = NONE; // the current scheduled item id
-    protected int _scheduleCount = 0; // the number of times the item has been delivered
+    protected int _scheduleCount = 0; // the number of times the item has been
+                                      // delivered
     protected int _reservedEnRoute = 0; // length of cars en route to this track
-    protected int _reservationFactor = 100; // percentage of track space for cars en route
+    protected int _reservationFactor = 100; // percentage of track space for
+                                            // cars en route
     protected int _mode = MATCH; // default is match mode
-    protected boolean _holdCustomLoads = false; // when true hold cars with custom loads
+    protected boolean _holdCustomLoads = false; // when true hold cars with
+                                                // custom loads
 
     // drop options
-    protected String _dropOption = ANY; // controls which route or train can set out cars
-    protected String _pickupOption = ANY; // controls which route or train can pick up cars
+    protected String _dropOption = ANY; // controls which route or train can set
+                                        // out cars
+    protected String _pickupOption = ANY; // controls which route or train can
+                                          // pick up cars
     public static final String ANY = "Any"; // track accepts any train or route
-    public static final String TRAINS = "trains"; // track only accepts certain trains // NOI18N
-    public static final String ROUTES = "routes"; // track only accepts certain routes // NOI18N
-    public static final String EXCLUDE_TRAINS = "excludeTrains"; // track excludes certain trains // NOI18N
-    public static final String EXCLUDE_ROUTES = "excludeRoutes"; // track excludes certain routes // NOI18N
+    public static final String TRAINS = "trains"; // track only accepts certain
+                                                  // trains // NOI18N
+    public static final String ROUTES = "routes"; // track only accepts certain
+                                                  // routes // NOI18N
+    public static final String EXCLUDE_TRAINS = "excludeTrains"; // track
+                                                                 // excludes
+                                                                 // certain
+                                                                 // trains //
+                                                                 // NOI18N
```

Comment:
Is this change intended? The change is that `// track excludes certain trains // NOI18N` is split on several lines.

There seems to be many similar changes in this file in this PR.